### PR TITLE
patches for ADL release

### PR DESCRIPTION
**Pull request by libinyang**
The patch includes:
1. sof-adl-rt711 support
2. revert dynamic pipeline feature

### CODE REVIEW

File: tools/topology/CMakeLists.txt [hunk 1]
```diff
@@ -20,16 +20,16 @@ set(TPLGS
...
-	"sof-hda-generic\;sof-hda-generic-3ch\;-DCHANNELS=4\;-DHSPROC=volume\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4\;-DDYNAMIC=1"
-	"sof-hda-generic\;sof-hda-generic-4ch\;-DCHANNELS=4\;-DHSPROC=volume\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4\;-DDYNAMIC=1"
-	"sof-hda-generic-kwd\;sof-hda-generic-2ch-kwd\;-DCHANNELS=2\;-DDYNAMIC=1"
-	"sof-hda-generic-kwd\;sof-hda-generic-4ch-kwd\;-DCHANNELS=4\;-DDYNAMIC=1"
-	"sof-hda-generic-idisp\;sof-hda-generic-idisp\;-DCHANNELS=0\;-DDYNAMIC=1"
-	"sof-hda-generic-idisp\;sof-hda-generic-idisp-2ch\;-DCHANNELS=2\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4\;-DDYNAMIC=1"
-	"sof-hda-generic-idisp\;sof-hda-generic-idisp-4ch\;-DCHANNELS=4\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4\;-DDYNAMIC=1"
```
Comment (ranj063) [inline tools/topology/CMakeLists.txt:32]: why are we reverting the whole commit. the hda-generic ones are just fine. If its the kwd pipelines that are problematioc, why not just revert that to not use dynamic pipelines until we fix it?

Comment (libinyang) [reply tools/topology/CMakeLists.txt:32]: @ranj063 It's not the kwd pipeline issue. We found if we enable dynamic pipeline, some hw params and intput/output errors will happen. Please see the details in https://github.com/thesofproject/linux/issues/2950.

**Comment by keyonjie** [comment]
@libinyang @kv2019i do we need the one for BT also:
```
68499a08baf2 topology: sof-tgl-rt711-rt1308: add BT offload for ADL
```

**Comment by libinyang** [comment]
> @libinyang @kv2019i do we need the one for BT also:
> 
> ```
> 68499a08baf2 topology: sof-tgl-rt711-rt1308: add BT offload for ADL
> ```

yes for BT

File: tools/topology/CMakeLists.txt (hunk 1 above)
Comment (lgirdwood) [reply tools/topology/CMakeLists.txt:32]: @libinyang please check with HEAD, this looks fixed by @keyonjie see my comment in the kernel thread.

Comment (libinyang) [reply tools/topology/CMakeLists.txt:32]: > @libinyang please check with HEAD, this looks fixed by @keyonjie see my comment in the kernel thread.

@lgirdwood This patch is for ADL release, not for main branch. I checked the recent daily test, dynamic pipeline works. I discussed with Mengdong and we won't apply the latest patches to ADL release branch this time.

**Comment by libinyang** [comment]
As ADL release uses another branch, I would like to close this PR.

**Pull request closed without merging**